src/components/Counter/Counter.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+class Counter extends React.Component {
+  state = {
+    count: 0,
```

Comment:
называй переменные существительными
```suggestion
    counter: 0,
```